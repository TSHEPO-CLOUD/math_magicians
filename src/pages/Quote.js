import React from 'react';
import './Quote.css';

export default function Quote() {
  return (
    <p className="quote">
      Do not worry too much about your difficulties in mathematics
      I can assure you that mine are still greater.
      Pure mathematics is, in its way, the poetry of logical ideas.
    </p>
  );
}
