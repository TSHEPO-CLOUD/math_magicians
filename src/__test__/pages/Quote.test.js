import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../../pages/Quote';

it('should match the snapshot of quote', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
