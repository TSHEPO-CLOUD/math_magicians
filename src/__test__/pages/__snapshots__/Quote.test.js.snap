// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot of quote 1`] = `
<p
  className="quote"
>
  Do not worry too much about your difficulties in mathematics I can assure you that mine are still greater. Pure mathematics is, in its way, the poetry of logical ideas.
</p>
`;
